### ISSUE TITLE
RStudio 1.2 compatibility

### ISSUE THREAD

**Issue by rogiersbart** [bkam235/ThemeSwitcher#3]
I just updated RStudio to the latest 1.2 version, which seems to break the addin (at least with R 3.5.3 and 3.6.0). Any idea if this can be fixed? According the release notes, they indeed expanded on the possibilities with respect to themes, so I suppose it is very likely the API has changed ... But I have no clue about the details there ...

**Comment by HulacSt** [comment]
Same here. Anyone know of any alternatives?

**Comment by HulacSt** [comment]
For anyone coming for an alternative, setting a couple of code snippets for  rstudioapi::applyTheme  has worked for me.

**Comment by rogiersbart** [comment]
Oh great, thanks for pointing to `rstudioapi::applyTheme()`! I just put this in my .Rprofile:
```r
light <- function() rstudioapi::applyTheme("Solarized Light")
dark <- function() rstudioapi::applyTheme("Solarized Dark")
```

**Comment by pat-s** [comment]
https://github.com/jannikbuhr/themeSwitch

**Comment by bkam235** [comment]
Yes, the API has changed. It has taken me a while to fix, but now it should work for R Studio 1.2 and be backwards compatible (still need to add sufficient tests).
